utils/threadpool/threadpool.h

```diff
@@ -225,6 +225,8 @@ class ThreadPool
         return fMaxThreads;
     }
 
+    inline size_t getIssuiedThreads() { return fIssued; }
```

Comment:
A typo to fix if you are going to change the code.

Reply:
Oh, sorry, thanks!